docker-compose.yml

```diff
@@ -66,13 +82,21 @@ services:
       - '9000:9000'
     volumes:
       - ch-data:/var/lib/clickhouse/
+    healthcheck:
+      test: wget --no-verbose --tries=1 --spider http://localhost:8123/ping || exit 1
+      interval: 2s
+      timeout: 2s
+      retries: 5
```

Comment:
I quite often get `dependency failed to start: container dialog_flow_framework-clickhouse-1 is unhealthy` in codespaces when running `docker compose`. Maybe we could increase these params?